src/App.js

```diff
@@ -114,7 +114,7 @@ function App(props) {
         />
       ),
     },
-    { path: "/*", element: <Viewer /> },
+    //{ path: "/*", element: <Viewer /> },
```

Comment:
With this commented out, the widget src can no longer come from the path. Unless no src is provided, props.src will override; is this intended?

I feel like that makes sense... -- but a reusable webcomponent like the livepeer wc may want to distribute a deploy with empty attributes, no default widget, for usage with bos-workspace -g flag

(thinking out loud)
Then index.html replacement could happen within the workspace from the app's bos.config.json, rather than here, in this repo, during web component build. A app only needs to handle the replacements in index.html then deploy this with the rest of the wc /dist dependency it used, for an easy app deploy to web4

Reply:
this test here does pass even with this change, and here the widget src comes from the path:

<img width="904" alt="image" src="https://github.com/NEARBuilders/near-bos-webcomponent/assets/9760441/f9541cbb-9caa-4502-ba9d-0e9c14a1d07b">